### ISSUE TITLE
Can share the log in the process of training video_renderer or give some guidance in the loss change?

### ISSUE THREAD

**Issue by lililuya** [Weizhi-Zhong/IP_LAP#46]
Hi,
Thanks for sharing code and your contribution in the field of talking head generation.
I am confused about the training loss of the video_renderer. I have trained the video_renderer in  LRS2 dataset which passed 24 epoches, 33220 steps, but the running_gen_loss seemed change randomly. The logs are as follow:
![image](https://github.com/Weizhi-Zhong/IP_LAP/assets/141640497/98f9fda9-d6db-4651-ac24-74cb875e34d9)
Is that normal or I did something wrong with that?  Looking for your reply! Thanks a lot!

**Comment by lililuya** [comment]
When the trainning came the 26 epoch,  step: 34560, the running_gen_loss dropped quickly. Just as following:
![image](https://github.com/Weizhi-Zhong/IP_LAP/assets/141640497/6619a644-2c80-4065-9d03-cf01a7e8ee67)
